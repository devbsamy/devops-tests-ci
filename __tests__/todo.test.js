const request = require('supertest');
const app = require('../src/index');

describe('GET /todos', () => {
  it('should return status 200 and an array', async () => {
    const res = await request(app).get('/todos');
    expect(res.statusCode).toBe(200);
    expect(Array.isArray(res.body)).toBe(true);
  });
  it('should match the todos snapshot', async () => {
  const res = await request(app).get('/todos');
  expect(res.body).toMatchSnapshot();
  });
});

