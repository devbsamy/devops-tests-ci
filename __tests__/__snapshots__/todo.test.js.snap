// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /todos should match the todos snapshot 1`] = `
[
  {
    "id": 1,
    "task": "Learn testing ðŸŽ¯",
  },
]
`;
